src/components/form/wizard/index.tsx

```diff
@@ -35,6 +36,15 @@ const Modal = styled.div`
     max-width: 1200px;
     z-index: 100;
     border-radius: 8px;
+    @media (max-width: 768px) {
```

Comment:
Can we use the one we have on `devices` ?

Reply:
okay